mono/io-layer/socket-wrappers.h

```diff
@@ -36,6 +36,7 @@
 #define _wapi_shutdown shutdown 
 #define _wapi_socket WSASocket 
 #define _wapi_select select 
+#define _wapi_cleanup_networking WSACleanup
```

Comment:
Well, actually now that I checked again I remember why I didn't call this: we already do it in `mono_networking_shutdown`.